code/modules/clothing/suits/jobs.dm

```diff
@@ -155,7 +155,18 @@
 	name = "surgical apron"
 	desc = "A sterile blue surgical apron."
 	icon_state = "surgical"
-	allowed = list(/obj/item/scalpel, /obj/item/surgical_drapes, /obj/item/cautery, /obj/item/hemostat, /obj/item/retractor)
+	allowed = list(
+		/obj/item/analyzer,
```

Comment:
```suggestion
		/obj/item/healthanalyzer,
```
This analyzer there is the Gas/Atmos one and not the health analyzer.

Reply:
Fixed.